packages/dev/storybook-builder-parcel/index.js

```diff
@@ -157,6 +158,16 @@ async function generateJS(options, overlayFS) {
   let dir = path.relative(__dirname, path.join(process.cwd(), stories[0].directory));
   let files = stories[0].files;
 
+  let storiesPath = path.join(dir, files);
+  const configImports = configs.map((config, i) => {
+    const relativePath = path.relative(__dirname, config);
+    return `import * as config_${i} from '${relativePath}';`;
+  });
+  let szConfigImports = configImports.join('\n');
+  if (isMingw()) {
+    storiesPath = storiesPath.replace(/\\/g, '/');
```

Comment:
rather than doing .replace, we could do something more like...
```
> path.posix.join('a','b')
'a/b'
> path.win32.join('a','b')
'a\\b'
```
so 
```
let curPath = path;
if (isMingw()) {
  curPath = path.posix;
}
```
and then all the "join"/"relative" things would work